cryodrgn/models.py

```diff
@@ -112,7 +112,7 @@ def load(self, config, weights=None, device=None):
             feat_sigma=c["feat_sigma"],
         )
         if weights is not None:
-            ckpt = torch.load(weights)
+            ckpt = torch.load(weights, map_location=device)
```

Comment:
Without the `map_location` argument, it becomes impossible to load a model trained on the GPU and run it on the CPU. After this change, I've tested this now with exactly such a scenario (loading a GPU-trained model on GPU/CPU), and it works for both cases.